pkg/apis/batch/validation/validation.go

```diff
@@ -815,6 +893,14 @@ func IsJobFailed(job *batch.Job) bool {
 	return IsConditionTrue(job.Status.Conditions, batch.JobFailed)
 }
 
+func IsJobSuccessCriteriaMet(job *batch.Job) bool {
```

Comment:
it looks like this doesn't have to be exported anymore.

Reply:
You're right.